### ISSUE TITLE
Disabling checks via feature command switches does not affect child zones

### ISSUE THREAD

**Issue by log1-c** [Icinga/icinga2#9845]
## Describe the bug

When using the feature command switches under /icingaweb2/health#!/icingaweb2/icingadb/health to disable host or service checks, I would expect them to stop completely. But they are still being executed
![image](https://github.com/Icinga/icingadb/assets/24474580/81e262aa-ee75-4515-a2da-90adffbe91f7)
![image](https://github.com/Icinga/icingadb/assets/24474580/e674e6e6-804e-4918-9689-f60ba8b42e6d)

## To Reproduce

Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include configuration, logs, etc. to reproduce, if relevant.

1. go to /icingaweb2/health#!/icingaweb2/icingadb/health
2. disable active checks

## Expected behavior

No checks are executed. 
## Your Environment

Include as many relevant details about the environment you experienced the problem in

* Icinga DB version: 1.0.1
* Icinga 2 version: 2.14.0
* Operating System and version: Ubuntu 22.04

**Comment by Al2Klimov** [comment]
We could stop processing checks from below nodes as long as they're disabled. 🙈

But jokes aside, IW2 actually just updates an attribute of a config object. And that runtime change is just synced inside the zone as usual. As long as not all Icinga 2 versions in a cluster forward such specific attribute updates below, we can only consider the option I described in the above paragraph. As a bridge technology.

**Comment by Tqnsls** [comment]
> We could stop processing checks from below nodes as long as they're disabled. 🙈

I don't think you intended to meant nothing else, but the config changes for the other buttons, especially the Event Handlers, should also be distributed to the child zones. We had to learn this the hard way that event handlers are not disabled globally by toggling the switch in IW2.

Is there another option to "globally" disable host/ service checks and eventhandlers without curl'ing the endpoints in the child zone?

**Comment by Al2Klimov** [comment]
💡 You could try putting the IcingaApplication in a global zone, maybe it will broadcast the updates then. But I don't know whether it works.
